lib/libutils/isoc/include/sys/_default_fcntl.h

```diff
@@ -0,0 +1,77 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/*
+ * Copyright (c) 1994-2009  Red Hat, Inc. All rights reserved.
+ */
+
+#ifndef	_SYS__DEFAULT_FCNTL_H_
+#ifdef __cplusplus
+extern "C" {
+#endif
+#define	_SYS__DEFAULT_FCNTL_H_
+#include <_ansi.h>
```

Comment:
Why is this file needed here? It defines a lot of macros that we don't use in OP-TEE. We should try to minimize the amount of legacy stuff introduced, considering that we're starting from scratch here.

Reply:
I removed `_default_fcntl.h` and unnecessary definitions.
Please see https://github.com/OP-TEE/optee_os/pull/6700/commits/062625aa6b4881b71f4d075ad6e7d80d0ba308fc